openedx/core/djangoapps/user_api/accounts/permissions.py

```diff
@@ -0,0 +1,20 @@
+"""
+Permissions classes for User accounts API views.
+"""
+from __future__ import unicode_literals
+
+from rest_framework import permissions
+
+
+class IsSuperUserOrCanDeactivateUser(permissions.BasePermission):
+    """
+    Grants access to AccountDeactivationView if the requesting user is a superuser
+    or has the explicit permission to deactivate a User account.
+    """
+    PERMISSION_TO_DEACTIVATE_USER_ACCOUNT_NAME = 'student.can_deactivate_users'
```

Comment:
Nit: this permission name will probably never change. Since it isn't being referenced anywhere besides the line below, I think it can go away.